dev-libs/boost/boost-1.88.0.ebuild

```diff
@@ -207,6 +205,122 @@ multilib_src_compile() {
 	fi
 }
 
+# TODO: multilib-minimal_abi_src_test is somehow not invoked
+src_test() {
+	# only use the best ABI
+	BUILD_DIR="${S}-$(echo $(multilib_get_enabled_abi_pairs) | awk '{print $NF}')"
+
+	# create custom options for tests based on the build settings
+	TEST_OPTIONS=("${OPTIONS[@]}")
+
+	# dial down log output - the full b2 command used to compile & run
+	# a test suite will be printed by ejam and can be used to build
+	# and run the tests in a test suite's directory.
+	TEST_OPTIONS=("${TEST_OPTIONS[@]/-d+2/-d0}")
+
+	# The following libraries do not compile or fail their tests:
+	local libs_excluded=(
+		# algorithm: fails to use std::reverse_copy
+		"algorithm"
+		# config: test output comparison failure
+		"config"
+		# contract: "C++03 support was deprecated in Boost.Chrono 1.82" ??
+		"contract"
+		# geometry: passes but takes >1 hour to build (enabled with USE=test-full)
+		"geometry"
+		# graph_parallel: tries to use mpi even with use=-mpi
+		"graph_parallel"
```

Comment:
Could it be tested under `if use mpi`?

Reply:
Handled in a separate if-use block. Thanks!